Documentation/cmdref/cilium-agent.md

```diff
@@ -248,6 +248,7 @@ cilium-agent [flags]
       --vtep-cidr strings                                    List of VTEP CIDRs that will be routed towards VTEPs for traffic cluster egress
       --vtep-endpoint strings                                List of VTEP IP addresses
       --vtep-mac strings                                     List of VTEP MAC addresses for forwarding traffic outside the cluster
+      --vtep-mask string                                     VTEP CIDR Mask for all VTEP CIDRs (default "255.255.255.0")
```

Comment:
New options should be documented in `Documentation/operations/upgrade.rst`, can you please add it?

Reply:
the first implementation is only released in pre-release candidate v1.12.0-rc0, , I am hoping after this PR is merged, there will be a final complete VTEP integration release in v1.12.0, updating upgrade.rst seems not needed, I  see `1.11 Upgrade Notes` and `1.12 Upgrade Notes` in `upgrade.rst`, the VTEP integration is not `1.12` yet.